m4/pdns_check_time_t.m4

```diff
@@ -1,4 +1,7 @@
 AC_DEFUN([PDNS_CHECK_TIME_T], [
+# _TIME_BITS=64 is supported on glibc-2.34 and requires _FILE_OFFSET_BITS=64
+CFLAGS="${CFLAGS} -D_FILE_OFFSET_BITS=64 -D_TIME_BITS=64"
```

Comment:
I do not like this unconditional approach much as it assumes that the whole world is glibc. Please add these defines only if needed.